<launch>
    <!--node pkg="serial_command_client" type="serial_command_client_node" name="serial_command_client_node_dev_ttyUSB0" output="screen">
        <param name="path_name" type="string" value="/dev/ttyUSB0"/>
    </node-->
    <!--node pkg="serial_command_client" type="serial_command_client_node" name="serial_command_client_node_dev_ttyUSB1" output="screen">
        <param name="path_name" type="string" value="/dev/ttyUSB1"/>
    </node-->
    <!--node pkg="serial_command_client" type="serial_command_client_node" name="serial_command_client_node_dev_ttyUSB2" output="screen">
        <param name="path_name" type="string" value="/dev/ttyUSB2"/>
    </node-->
    <!--node pkg="berry_imu" type="berry_imu_node" name="berry_imu_node" output="screen">
        <param name="path_name" type="string" value="/dev/ttyUSB0"/>
        <param name="imu_topic" type="string" value="/imu/data_raw"/>
        <param name="mag_topic" type="string" value="/imu/mag"/>
    </node-->
    <!--node pkg="imu_filter_madgwick" type="imu_filter_node" name="imu_filter_node">
        <param name="use_magnetic_field_msg" type="bool" value="true"/>
        <param name="use_mag" type="bool" value="true"/>
        <param name="publish_tf" type="bool" value="false"/>
        <param name="world_frame" type="string" value="enu"/>
    </node-->
    <!--node pkg="coverage_path_planner" type="coverage_path_planner_node" name="coverage_path_planner_node" output="screen">
        <param name="make_plan_service" type="string" value="/core/make_plan"/>
    </node-->
    <node pkg="adr" type="adr_node" name="adr_node" output="screen">
        <param name="set_state_service" type="string" value="/adr/set_state"/>
        <param name="get_state_service" type="string" value="/adr/get_state"/>
    </node>
    <node pkg="rest" type="rest_node" name="rest_node" output="screen">
    </node>
</launch>